bears/general/InvalidLinkBear.py

```diff
@@ -40,22 +43,28 @@ def get_status_code(url, timeout):
             pass
 
     @staticmethod
-    def find_links_in_file(file, timeout, link_ignore_regex):
+    def find_links_in_file(file, network_timeout, link_ignore_regex):
         link_ignore_regex = re.compile(link_ignore_regex)
         regex = re.compile(
-            r'((ftp|http)s?://[^.:%\s_/?#[\]@\\]+\.(?:[^\s()%\'"`<>|\\]+|'
-            r'\([^\s()%\'"`<>|\\]*\))*)(?<!\.)(?<!,)')
+            r'((ftp|http)s?://[^.:\s_/?#[\]@\\]+\.(?:[^\s()\'"`<>|\\]+|'
+            r'\([^\s()\'"`<>|\\]*\))*)(?<!\.)(?<!,)')
```

Comment:
seems you have rebased but kept the old changes :3
Please remove this change, we want the new regex :+1: